assemble/conf/log4j-monitor.properties

```diff
@@ -15,18 +15,18 @@
 
 # Write out everything at the DEBUG level to the debug log
 log4j.appender.A2=org.apache.log4j.RollingFileAppender
-log4j.appender.A2.File=${org.apache.accumulo.core.dir.log}/${org.apache.accumulo.core.application}_${org.apache.accumulo.core.ip.localhost.hostname}_fromprops.debug.log
+log4j.appender.A2.File=${accumulo.log.dir}/${accumulo.application}_${accumulo.local.hostname}.debug.log
```

Comment:
Most the properties are being set in accumulo-env.sh.  Some of the log forwarding properties are set in Java but that should change with your new appender.